ee/clickhouse/queries/funnels/funnel.py

```diff
@@ -15,45 +15,54 @@ def get_query(self, format_properties):
         steps_per_person_query = self.get_step_counts_query()
         max_steps = len(self._filter.entities)
 
+        breakdown_clause = self._get_breakdown_prop()
+
         return f"""
-        SELECT {self._get_count_columns(max_steps)} {self._get_step_time_avgs(max_steps)} FROM (
+        SELECT {self._get_count_columns(max_steps)} {self._get_step_time_avgs(max_steps)} {breakdown_clause} FROM (
                 {steps_per_person_query}
-        ) SETTINGS allow_experimental_window_functions = 1
+        ) {'GROUP BY prop' if breakdown_clause != '' else ''} SETTINGS allow_experimental_window_functions = 1
```

Comment:
```suggestion
        ) {f'GROUP BY {breakdown_clause}' if breakdown_clause != '' else ''} SETTINGS allow_experimental_window_functions = 1
```

Reply:
I'm guessing you didn't like this, may I know why? 😅

Reply:
ah ffs, I see, `breakdown_clause` has the leading comma 🤦 . Oh well, I'll put this in the "do when all commas are removed" folder.